src/test/java/org/agbrothers/englishbot/processing/processor/lesson/english/EnglishAudioLessonProcessorTest.java

```diff
@@ -0,0 +1,28 @@
+package org.agbrothers.englishbot.processing.processor.lesson.english;
+
+import org.agbrothers.englishbot.entity.Word;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class EnglishAudioLessonProcessorTest {
+
+    @Test
+    void getValueToTranslate() {
+        EnglishAudioLessonProcessor englishAudioLessonProcessor = new EnglishAudioLessonProcessor();
+        Word word = new Word("word", "слово");
+
+        String result = englishAudioLessonProcessor.getValueToTranslate(word);
+
+        assertEquals("", result);
+
+    }
+
+    @Test
+    void process() {
+    }
+
+    @Test
+    void setRestTemplate() {
```

Comment:
no tests for setters. Please doublecheck your code before sending it to review

Reply:
added test for this method